packages/@expo/cli/src/start/index.ts

```diff
@@ -68,9 +69,13 @@ export const expoStart: Command = async (argv) => {
         `--offline                              Skip network requests and use anonymous manifest signatures`,
         `--https                                Start the dev server with https protocol`,
         `--scheme <scheme>                      Custom URI protocol to use when launching an app`,
-        chalk`-p, --port <port>                      Port to start the dev server on (does not apply to web or tunnel). {dim Default: 19000}`,
+        chalk`-p, --port <port>                      Port to start the dev server on (does not apply to web or tunnel). {dim Default: 8081}`,
         ``,
-        chalk`--dev-client                           {yellow Experimental:} Starts the bundler for use with the expo-development-client`,
+        `--app-launch-mode <mode>               Override the app launch mode. If not specified, the app will try to determine the best launch mode based on the current project dependencies and workflow.`,
+        chalk`                                       {bold start}: Directly start the app`,
+        chalk`                                       {bold open-deep-link-dev-client}: Launch expo-development-client with deep link`,
+        chalk`                                       {bold open-deep-link-expo-go}: Launch Expo Go with deep link`,
+        chalk`                                       {bold open-redirect-page}: Open the redirect page in browser`,
```

Comment:
Suggestion to answers questions developers may have in order to help them decide which choice to select.
```suggestion
        chalk`                                       {bold start}: Directly start the app`,
        chalk`                                       {bold dev-client}: Launch the app's dev client with a deep link. expo-dev-client must be installed and built into the app.`,
        chalk`                                       {bold expo-go}: Launch Expo Go with a deep link`,
        chalk`                                       {bold browser}: Open your browser to a web page with links to launch the app in different modes`,
```